pkg/server/options/options.go

```diff
@@ -97,7 +97,7 @@ func NewOptions() *Options {
 	}
 
 	// override all the stuff
-	o.GenericControlPlane.SecureServing.ServerCert.CertDirectory = "" // will be completed to be relative to root dir
+	o.GenericControlPlane.SecureServing.ServerCert.CertDirectory = rootDir
```

Comment:
@ncdc this

Reply:
have not understood yet why I did this strange logic. Wished my former me would have commented better 🤷‍♂️

Reply:
Maybe the root-directory flag was the reason. But now it is much clearer.